stix2validator/output.py

```diff
@@ -149,25 +149,43 @@ def print_results(results):
         results: A list of FileValidationResults instances.
 
     """
-    for file_result in sorted(results, key=operator.attrgetter("filepath")):
-        print_horizontal_rule()
-        print_level(logger.info, "[-] Results for: %s", 0, file_result.filepath)
-
```

Comment:
This pattern of checking types is sometimes necessary to avoid circular imports, but it feels really clunky and is a bit of a code smell to me. It might be better to define a `print()` method on `ObjectValidationResults` and `FileValidationResults`, which each delegate to separate `print_object_results` and `print_file_results` functions in this module. Just an idea.

Reply:
I like this idea; hopefully my latest commits have less code smell. I called it `log()` instead though, because functions can't be called `print()` in Python 2.